src/index.js

```diff
@@ -0,0 +1,206 @@
+    this.completed = completed;
+    this.index = index;
+  }
+}
+
+let toDos = [];
+
+function createTodoItem(todo) {
+  const li = document.createElement('li');
+  li.innerHTML = `
+    <div class="flex todo-element">
+      <div>
+          <input type="checkbox" class="checkbox"
+          ${todo.completed ? 'checked' : ''}>
+          <span>${todo.description}</span>
+      </div>
+      <span class="material-icons edit-icon" style="cursor: pointer">
+          more_vert
+      </span>
+    </div>
+    <hr>`;
+  return li;
+}
+
+function ReplaceTodoItem(todo) {
+  const edit = `
+    <div>
+      <input type="checkbox" class="checkbox" 
+      ${todo.completed ? 'checked' : ''}>
+      <span>${todo.description}</span>
+    </div>
+    <span class="material-icons edit-icon" style=" cursor: pointer">
+        more_vert
+    </span>
+      `;
+  return edit;
+}
+
+function addTodoItem(todo) {
+  const li = createTodoItem(todo);
+  button.parentElement.insertBefore(li, button);
+}
+
+function todoItem() {
+  toDos.sort((a, b) => (a.index > b.index ? 1 : -1));
+  toDos.forEach((todo) => {
+    addTodoItem(todo);
+  });
+}
+
+function saveTodosLocally() {
+  localStorage.setItem('todos', JSON.stringify(toDos));
+}
+
+function ReplaceTodoItemForCompletedTask(todo) {
+  const edit = `
+  
+  <div>
+  <span class="material-icons edit-icon" style=" cursor: pointer; color: green">
+      done
+  </span>
+    <strike><span>${todo.description}</span></strike>
+  </div>
+  <span>
+  You complete the task!<br>clear done's to remove me!
+  </span>
+    `;
+
+  return edit;
+}
+
+function changeElementToCompleted(index) {
+  update(toDos[index]);
+  saveTodosLocally();
+  if (toDos[index].completed) {
+    const completedElement = ReplaceTodoItemForCompletedTask(toDos[index]);
+    const todoElements = document.querySelectorAll('.todo-element');
+    todoElements[index].innerHTML = completedElement;
+  }
+}
+
+function addEventsToCheckboxes(recievedIndex) {
+  const checkboxes = document.querySelectorAll('.checkbox');
+  checkboxes.forEach((checkbox, index) => {
+    if (recievedIndex) {
+      if (recievedIndex === index) {
+        checkbox.addEventListener('change', () => {
+          changeElementToCompleted(index);
+        });
+      }
+    } else {
+      checkbox.addEventListener('change', () => {
+        changeElementToCompleted(index);
+      });
+    }
+  });
+}
+
+function addEventsToEditIcons() {
+  const editIcons = document.querySelectorAll('.edit-icon');
+  const todoElements = document.querySelectorAll('.todo-element');
+
+  toDos.forEach((todo, index) => {
+    editIcons[index].addEventListener('click', () => {
+      const div = document.createElement('div');
+      div.classList.add('flex', 'todo-element');
+      div.style.backgroundColor = '#FFFBAE';
+
+      const checkbox = document.createElement('input');
+      checkbox.type = 'checkbox';
+      checkbox.classList.add('checkbox');
+      checkbox.checked = todo.completed;
+
```

Comment:
You can keep your code DRY by not repeating your code more than once.
you can do that by creating one function to add the new to-do task to the DOM and call it where ever you want.